.github/workflows/acceptance-tests.yml

```
@@ -0,0 +1,48 @@
+name: Acceptance Tests
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - .github/workflows/acceptance-tests.yml
+      - 'logging-k8s-stdout-otlp-json/**'
+  workflow_dispatch:
+
+jobs:
+  acceptance-tests:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Check out
+        uses: actions/checkout@v4
+
+      - name: Set up JDK for running Gradle
+        uses: actions/setup-java@v4
+        with:
+          distribution: temurin
+          java-version: 17
+
+      - name: Set up gradle
+        uses: gradle/actions/setup-gradle@v4
+        with:
+          cache-read-only: ${{ github.event_name == 'pull_request' }}
+
+      - name: Check out oats
+        uses: actions/checkout@v4
+        with:
+          repository: grafana/oats
+          ref: bc2f08c5e55114234cece216290f1580a75a6032
+          path: oats
+      - name: Set up Go
```

(sorry, just a nitpicky preference I have)
```suggestion

      - name: Set up Go
```